### PR TITLE
Update with-office-365.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6503
Added note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/with-office-365.md [hunk 1]
```diff
@@ -141,6 +141,9 @@ For details on Skype for Business Online Plans, see the [Skype for Business Onli
    Set-AzureADUser -UserPrincipalName <Account> -PhoneNumber "<PhoneNumber>"
    ```  -->
 
+> [!NOTE]
+> If the password is not set to Never Expire, then upon the account reaching the expiry period, the account will no longer sign-in on the device. The password will then need to be changed for the account and also updated locally on the MTR device.
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-office-365.md:145]:
```suggestion
> If the password is not set to Never Expire, the account will no longer sign-in on the device when the account reaches the expiry period. The password will then need to be changed for the account and also updated locally on the MTR device.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

File: Teams/rooms/with-office-365.md [hunk 2]
```diff
@@ -141,6 +141,9 @@ For details on Skype for Business Online Plans, see the [Skype for Business Onli
    Set-AzureADUser -UserPrincipalName <Account> -PhoneNumber "<PhoneNumber>"
    ```  -->
 
+> [!NOTE]
+> If the password is not set to Never Expire, the account will no longer sign-in on the device when the account reaches the expiry period. The password will then need to be changed for the account and also updated locally on the MTR device.
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-office-365.md:145]:
```suggestion
> If the password is not set to Never Expire, the account will no longer sign in on the device when the account reaches the expiry period. The password will then need to be changed for the account and also updated locally on the MTR device.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies Lindsay - one more please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**